src/iop/watermark.c

```diff
@@ -162,6 +178,48 @@ int legacy_params(dt_iop_module_t *self, const void *const old_params, const int
     n->rotate = 0.0;
     n->sizeto = DT_SCALE_IMAGE;
     g_strlcpy(n->filename, o->filename, sizeof(n->filename));
+    g_strlcpy(n->text, _("text"), sizeof(n->text));
```

Comment:
What is the point of storing _("text") in updated legacy entry?
I do understand that it will not show that line on each upgraded image, but plain "" would be enough IMO.